task/sast-coverity-check-oci-ta/0.2/sast-coverity-check-oci-ta.yaml

```diff
@@ -349,6 +349,12 @@ spec:
           fi
         fi
 
+        # avoid tracing "dnf --installroot" because it would break the build due to unreachable /tmp/idir in the chroot
+        export COVERITY_DISENGAGE_EXES="dnf"
```

Comment:
Wondering if this should also include `microdnf` and `yum`?

 Also, do you know if coverity will combine this with the default value? I.e. if coverity would have set it to "qemu;ldd", and we set it to "dnf", will the final value used at runtime be "qemu;ldd;dnf"?